packager/media/codecs/h264_byte_to_unit_stream_converter_unittest.cc

```diff
@@ -79,5 +79,50 @@ TEST(H264ByteToUnitStreamConverter, ConversionFailure) {
   EXPECT_FALSE(converter.GetDecoderConfigurationRecord(&decoder_config));
 }
 
+TEST(H264ByteToUnitStreamConverter, NaluConversionWithSpsExtension) {
+  const uint8_t kSpsExtArr[] = {
```

Comment:
It is not correct to call it `kSpsExtArr`. Can you name it kByteStreamWithSpsExtension?

Reply:
done